layouts/shortcodes/docs/languages/libraries-intro.md

```diff
@@ -0,0 +1,48 @@
+<!-- prettier-ignore -->
+{{ $lang := .Get 0 -}}
+{{ $howMany := .Get 1 | default 10 -}}
+{{ $integrations := where (where $.Site.Data.ecosystem.integrations ".components" "intersect" (slice $lang)) ".native" "eq" true -}}
+
+When you develop an app, you make use of third-party libraries and frameworks to
+accelerate your work and to not reinvent the wheel. If you now instrument your
+app with OpenTelemetry, you don't want to spend additional time on manually
+adding traces, logs and metrics to those libraries and frameworks. Fortunately,
+you don't have to reinvent the wheel for those either: libraries might come with
+OpenTelemetry support natively or you can use an
+[Instrumentation Library](/docs/concepts/instrumentation/libraries/) in order to
+generate telemetry data for a library or framework.
+
+If you are instrumenting an app, you can learn on this page how to make use of
+natively instrumented libraries and Instrumentation Libraries for your
+dependencies.
+
+## Use natively instrumented libraries
+
+If a library comes with OpenTelemetry out of the box, you get the traces,
+metrics and logs emitted from that library, by adding and setting up the
+OpenTelemetry SDK with your app.
+
+The library may provide some additional configuration for the instrumentation.
+Go to the documentation of that library to learn more.
```

Comment:
```suggestion
## Use OTel-ready libraries

If a library comes with OpenTelemetry support by default, you can get
traces, metrics, and logs emitted from that library by adding and setting up
the OpenTelemetry SDK with your app.

The library might require some additional configuration for the instrumentation.
Go to the documentation of that library to learn more.
```

IMHO, since the meaning of "native" is not exactly clear outside of OTel, we could use a clearer adjective, like the compound "OTel-ready".

Reply:
Fair point, but we might extract this discussion out, because there are a few more places where we then would need to change it. 

So for this PR let's stick with "native", and raise a new PR for having that discussion, wdyt?

Reply:
Agreed! I've changed my suggestion accordingly. Shall I raise an issue or a PR for the native thing?

Reply:
let's kick this of with an issue, there are a lot of places to fix that, so we should agree on a term first and then go forward